patterns/ecs-v1/linux-syslog

```diff
@@ -1,16 +1,16 @@
 SYSLOG5424PRINTASCII [!-~]+
 
-SYSLOGBASE2 (?:%{SYSLOGTIMESTAMP:timestamp}|%{TIMESTAMP_ISO8601:timestamp8601}) (?:%{SYSLOGFACILITY} )?%{SYSLOGHOST:logsource}+(?: %{SYSLOGPROG}:|)
-SYSLOGPAMSESSION %{SYSLOGBASE} (?=%{GREEDYDATA:message})%{WORD:pam_module}\(%{DATA:pam_caller}\): session %{WORD:pam_session_state} for user %{USERNAME:username}(?: by %{GREEDYDATA:pam_by})?
+SYSLOGBASE2 (?:%{SYSLOGTIMESTAMP:timestamp}|%{TIMESTAMP_ISO8601:timestamp}) (?:%{SYSLOGFACILITY} )?%{SYSLOGHOST:[host][hostname]}+(?: %{SYSLOGPROG}:|)
```

Comment:
If I remember correctly, the convention is to shove the date into `timestamp8601`, parse it with the date filter, which then populates `@timestamp`. Since `timestamp` is not an ECS field, from ECS' point of view, renaming this field is not necessary.

If you're doing this for other reasons such as aligning various grok patterns to use `timestamp` as an interim field instead, that's also fine. I'm just pointing out that this isn't required per ECS.

Reply:
thanks, it's mostly only to use one `timestamp` - to align with the conventions with other patterns.
this is [handled](https://github.com/logstash-plugins/logstash-input-syslog/blob/v3.4.4/lib/logstash/inputs/syslog.rb#L293-L294) at the input-syslog plugin (before sending it to a date parser) - won't mind having a single `timestamp` : 
`event.set("timestamp", event.get("timestamp8601")) if event.include?("timestamp8601")`

as this is still a draft having these name-spaced `system.syslog.timestamp` is an open option ...
but I think for users the less work there will be to migrate the better so `timestamp` likely wins that battle.